src/core/utils/StringUtilities.h

```diff
@@ -33,9 +34,7 @@ namespace StringUtilities {
                                                         const std::string  &hangingPad,
                                                         size_t              screenWidth);                           //!< Format string for output to screen
     std::string                 formatTabWrap(std::string s, size_t tabs, size_t width, bool removeFormat=true);    //!< Wraps texts.
-    std::string                 getFileContentsAsString(const std::string& s);                                      //!< Convert the file contents to a string
-    std::string                 getStringWithDeletedLastPathComponent(const std::string& s);                        //!< Convert the file contents to a string
-    std::string                 getLastPathComponent(const std::string& s);                                         //!< Find the last component of a file path
```

Comment:
They not used anymore.  They are also unneeded, because path.filename() gets the last path component, and path.parent_path() gets the path without the last path component.